docs/documentation/features/rules.md

```diff
@@ -107,9 +107,9 @@ The types in the **Configuration** object are restricted to the following:
 
 **JSON schemas for:**
 
- * [module types](../../../bundles/automation/org.eclipse.smarthome.automation.json/src/main/resources/ModuleTypeDefinitions_JSONschema.json)
- * [rule templates](../../../bundles/automation/org.eclipse.smarthome.automation.json/src/main/resources/TemplateDefinitions_JSONschema.json)
- * [rule instances](../../../bundles/automation/org.eclipse.smarthome.automation.json/src/main/resources/RuleDefinitions_JSONschema.json)
+ * [module types](../../../docs/documentation/development/rules/ModuleTypes_schema.md)
```

Comment:
I would keep the links to the pure sjon files, so that people can directly use them, if they want.
Otherwise, you would have to set the links to .html (because the generated website has html files) and you would need to add the Jekyll header and footer to the md files.